server/spec/requests/reviews_spec.rb

```diff
@@ -0,0 +1,131 @@
+require 'rails_helper'
+
+RSpec.describe 'Reviews API', :with_reviews, type: :request do
+  let!(:restaurant) { create(:restaurant) }
+  let!(:reviews) { create_list(:review, non_zero_rand(20), restaurant: restaurant) }
+
+  context 'GET /restaurant/:restaurant_id/reviews' do
```

Comment:
generally for this test file: we usually want the contexts to be a phrase that let us know what is being tested a `when` sentence or an `and` sentence if the context are nested, and each tested method is under a separate `describe`.
for example:
```suggestion
  describe '#index' do
       context 'when a restaurant id is given' do
          context 'and it has reviews' do...
```

Reply:
also you only tested the happy paths, please add tests for where the `restaurant_id` given does not exists or the restaurant does not have any reviews etc